docs/persistent-storage.md

```diff
@@ -0,0 +1,120 @@
+# Set up Persistent Volume Claim for file storage
+
+Persistent storage allows data to be available even if the container, pod, or the worker node is removed.  This data can be stored in either a file system or a database.  This guide walks through setting up a persistent storage for a file system, using Persistent Volume Claim (PVC) on the Kubernetes cluster.  This involves creating a PVC resource, and mounting volume storage on the deployment to the cluster.  We will first set up the PVC configuration in `values.yaml` file for the Helm PVC and deployment template to use values from.  Next we'll define the PVC template and modification to the deployment template.
+
+## 1. Define PVC configuration in `values.yaml`
+
+In the `values.yaml` file, located in the folder `chart/app-name`, add a section for `pvc` containing the configuration of PVC for the cluster:
+
+```
+pvc:
+  volName: <volume name>
+  pvcName: <pvc name>
+  mountPath: <mount path>
+  labels:
+    billingType: <billing type>
+    region: <region>
+    zone: <zone>
+  spec:
+    accessModes:
+      - <access modes>
+    resources:
+      requests:
+        storage: <storage>
+    storageClassName: <storage class name>
+```
+
+Update these fields as the following:
+* `<volume name>`:  The name of the volume to mount to your pod.
+* `<pvc name>`:  The name of the PVC resource.
+* `<mount path>`:  The absolute path of the directory to persist as file storage.
+* `<billing type>`:  The frequency for which your storage bill is calculated, either `monthly` or `hourly`.
+* `<region>`:  The region where you want to provision your file storage.
+* `<zone>`:  The zone where you want to provision your file storage.
+* `<access modes>`:  This should be one of the following options: `ReadWriteMany`, `ReadOnlyMany`, `ReadWriteOnce`.
+* `<storage>`:  The size of the file storage, in gigabytes (Gi).
+* `<storage class name>`:  The name of the volume to mount to your pod. You can choose to use one of the [IBM-provided storage classes](https://cloud.ibm.com/docs/containers?topic=containers-file_storage#file_storageclass_reference) or [create your own storage class](https://cloud.ibm.com/docs/containers?topic=containers-file_storage#file_custom_storageclass). This creates an instance of IBM Cloud File Storage to be used by the Kubernetes PV.
+
```

Comment:
delete all the above field descriptions as they are duplicates (see above comment)